src/test/java/guitests/MenuControlTest.java

```diff
@@ -117,6 +117,9 @@ public void menuControlTest() {
 
         // Testing First Panel selected
         assertEquals(0, (int) panelControl.getCurrentlySelectedPanel().get());
+        press(KeyCode.CONTROL).press(KeyCode.DIGIT1).release(KeyCode.DIGIT1).release(KeyCode.CONTROL);
+        assertEquals(0, (int) panelControl.getCurrentlySelectedPanel().get());
+        // TODO (currently not possible) test that panelsScrollPane.getHValue() is 0
```

Comment:
ScrollPane originates from UI, but it doesn't have a variable that stores it. It stores one that is the grandparent of the ScrollPane, hence accessing it from UI will be quite convoluted. So in this case, MenuControl actually have direct reference to ScrollPane (under the variable `panelsScrollPane`.

This test is added to MenuControlTest where they do panel menu testing and board menu testing. I added it to OpenBoard testing. The test for panel focus is on another file (PanelFocusTest). This is because I test the panel focus for 2 cases: on startup and on opening a board.